www/pages/blog/release/v0-23-0.md

```diff
@@ -0,0 +1,120 @@
+---
+label: 'blog'
+title: v0.23.0 Release
+template: blog
+---
+
+# Greenwood v0.23.0
+
+## What's New
+
+With this new release, the Greenwood team is excited to (soft) launch the ability to add Server Side Rendering (SSR) to your Greenwood project as well as support for using a custom renderer like [**Lit** SSR](https://www.npmjs.com/package/@lit-labs/ssr).  Additionally, to enhance the ability of purely static sites to benefit from some build time templating, a new feature called "interpolate frontmatter" was introduced to easily reuse frontmatter similar to how you would use JavaScript interpolation, but in your HTML and markdown.  Let's highlight them both below!  👇
+
+## Server Side Rendering (SSR)
+
+As mentioned above, we are soft launching the ability to incorporate server rendering into your Greenwood projects.  By simply adding a JavaScript file to your project, you will be able to have server rendered content available when running `greenwood serve`.  You can also combine static and server rendered content all in the same project for a hybrid application!  Let's take a look at a quick example.
+
+### How It Works
+You can add a file to your project in the _pages/_ directory and implement either of the three supported APIs, and you will have a server rendered route available!
+```shell
+.
+└── src
+    └── pages
+        ├── artists.js
+        ├── about.md
+        └── index.html
+```
+
+```js
+// artists.js
+import fetch from 'node-fetch'; // this needs to be installed from npm
+
+async function getBody(compilation) {
+  const artists = await fetch('http://www.example.com/api/artists').then(resp => resp.json());
+
+  return `
+    <body>
+      <h1>Hello from the server rendered artists page! 👋</h1>
+      <table>
+        <tr>
+          <th>Name</th>
+          <th>Image</th>
+        </tr>
+        ${
+          artists.map((artist) => {
+            const { name, imageUrl } = artist;
+            return `
+              <tr>
+                <td>${name}</td>
+                <td><img src="${imageUrl}"/></td>
+              </tr>
+            `;
+          });
+        }
+      </table>
+    </body>
+  `
+}
+
+export { getBody }
+```
+
+You can then access `/artists/` and see the content!  💥
+
+
+![Server Side Rendering example](/assets/blog-images/ssr.webp)
+
+> _In the above screenshot, we can also see a demonstration of custom rendering using LitSSR with the `<simple-greeting>` component._
+
+## Interpolate Frontmatter
+At the risk of implementing / using a custom templating system (e.g. handlebars, nunjucks, etc), which is something Greenwood strives to avoid, but still recognizing that requiring JavaScript though our _graph.json_ for static sites can be a bit cumbersome.  With this new feature, which is a flag that can be enabled in your _greenwood.config.js_, frontmatter in your markdown will be available in your HTML or markdown.  Great for `<meta>` tags!
```

Comment:
This could be improved as it went a few directions quickly and I was lost while reading.